CompilerSource/compiler/jdi_utility.cpp

```diff
@@ -132,6 +132,8 @@ bool lang_CPP::global_exists(string n) const {
 void lang_CPP::quickmember_variable(jdi::definition_scope* scope, jdi::definition* type, string name) {
   scope->members[name] = std::make_unique<jdi::definition_typed>(name,scope,type);
 }
+
+enigma::parsing::StdErrorHandler hackybaby;  // TODO: FIXME: This should be using a central error handler...
```

Comment:
More or less exactly what we do elsewhere (e.g. `lang_CPP.cpp`), except we keep it in the function instead of making it a global.

```cpp
   enigma::parsing::StdErrorHandler hack;  // TODO: FIXME: This should be using a central error handler...
```